docs/components/connectors/react-components/_connectors-card-sml.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./_connectors-table.css";
+
+const ConnectorCard = ({ link, title, image, description }) => {
+  return (
+    <a href={link} className="connector-small-link" title={title}>
+      <div className="connector-small">
+        <img
+          src={image}
+          alt={`${title} icon`}
+          className="connector-small-image"
+        />
+        {title}
+      </div>
+    </a>
+  );
+};
+
+const ConnectorsGridSml = ({ connectors }) => {
```

Comment:
non-blocking: renaming this `ConnectorsGridSmall` and the file `connectors-card-small` would add clarity (I had to take some time to think what sml stood for 😅).